app/views/community_memberships/_confirmation_form.haml

```diff
@@ -2,11 +2,14 @@
   initialize_confirmation_pending_form("#{I18n.locale}","#{email_not_accepted_message}");
 
 - email_to_confirm = @current_user.latest_pending_email_address(@current_community)
+- contact_support_link = link_to t("sessions.confirmation_pending.contact_support_link_text"), "mailto:#{support_email}", "data-uv-trigger" => "contact"
```

Comment:
Oh, yea that makes sense. It didn't do it just by adding the attribute, so I guess some JS needs to be prepared too.
On the other hand in this particular case a direct mailto might even be better, as if there are issues with mail confirmation, the knowledge base can't help much more than the "check spam filter" instruction already on the page.. So I might now just leave the attribute out for simplicity, and to encourage trials starters who are in trouble at this stage to speak to a human. :)